Documentation/benchmarks/rkt-1-4-0-benchmarks.md

```diff
@@ -17,6 +17,10 @@ rkt(18493): seconds alive: 10  avg CPU: 28.314541%  avg Mem: 2 mB  peak Mem: 2 m
 systemd(18515): seconds alive: 9  avg CPU: 0.000000%  avg Mem: 4 mB  peak Mem: 4 mB
 systemd-journal(18517): seconds alive: 9  avg CPU: 88.397098%  avg Mem: 7 mB  peak Mem: 7 mB
 worker(18521): seconds alive: 9  avg CPU: 7.330367%  avg Mem: 5 mB  peak Mem: 6 mB
+load average in a container: Load1: 0.390000 Load5: 0.120000 Load15: 0.080000
```

Comment:
I just mean the format of the string here https://github.com/coreos/rkt/pull/2587/files#diff-07b54b89d53b286f0316e52cfcec59afR183 is different, which suggests that you artificially created these outputs rather than running the code ;-)